deployment/BUILD

```diff
@@ -39,3 +39,12 @@ write_flag_to_file(
     flag = ":image_tag",
     visibility = ["//visibility:public"],
 )
+
+# TODO: rules OCI only support tag file names with the ".txt" extension.
+genrule(
```

Comment:
Let's use skylib `copy_file` with `allow_symlink` instead for better RBE integration.

Reply:
Hmm I do like `copy_file` but I don't think it's better for RBE integration.

See https://github.com/bazelbuild/bazel-skylib/blob/e8e9d218ff563f93e3a7f7547ea532e66b210620/rules/private/copy_common.bzl#L43
and https://github.com/bazelbuild/bazel-skylib/issues/533 for more information.

Reply:
These exec properties should only be used when the rule runs a non-internal action, but with `allow_symlink` it just runs the internal symlink action which doesn't even take exec properties. That's why I think it's going to perform better.

Reply:
ah right. We have `allow_symlink=True` here already. Good call!